client/src/redux/index.js

```diff
@@ -43,8 +43,10 @@ export const defaultDonationFormState = {
 
 const initialState = {
   appUsername: '',
+  showMultipleProgressModals: false,
   recentlyClaimedBlock: null,
-  canRequestProgressDonation: true,
+  completionCountWhenShownProgressModal: null,
```

Comment:
```suggestion
  completionCountWhenShownProgressModal: 0,
```
0 seems slightly nicer, but it's fine either way.